src/Symfony/Bundle/TwigBundle/ContainerAwareRuntimeLoader.php

```diff
@@ -11,8 +11,8 @@
 
 namespace Symfony\Bundle\TwigBundle;
 
+use Psr\Container\ContainerInterface;
 use Psr\Log\LoggerInterface;
```

Comment:
updated to use the twig class, it will break tests until https://github.com/symfony/symfony/pull/21625 is merged and released